pom.xml

```diff
@@ -92,6 +92,18 @@
       <artifactId>throttle-concurrents</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter</artifactId>
+      <version>5.10.0</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-inline</artifactId>
+      <version>4.0.0</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
I'm unwilling to add a dependency on the outdated mockito-inline library when it has been replaced with mockito-core.  If you must use mock objects, please use mockito-core.  Since there are no other tests in the plugin based on mock objects, I'd very much prefer that this test also not use mock objects.
```suggestion
```

Reply:
Removed the use of mock objects entirely.

Reply:
That's great!  Thanks very much for adapting to my comments, especially when they were very general.